### PR TITLE
Dg seo updates

### PR DESCRIPTION
**Pull request by Dgiordano33**
## Affected Components
* [ ] Content & Marketing
* [ ] Pricing
* [ ] Test
* [ ] Docs
* [ X] Learn
* [ ] Other

## Pre-Requisites
* [ ] Code is linted (`npm run lint`)

## Notes for the Reviewer
We updated the structure and content of our Learn section to focus on "Playwright" in this phase. 
- Removed mentions of Puppetter
- Improved the structure of the sidebar nav
- Optimized H1s, Titles, and URLs
- Built redirects to move traffic over from the old URL

## Screenshots

image

### CODE REVIEW

File: site/content/docs/monitoring/checks/_index.md
Comment (miliberlin) [inline site/content/docs/monitoring/checks/_index.md:1]: @Dgiordano33 Did you mean to commit this? You just added a blank page
Screenshot 2024-09-27 at 13 31 59

**Review by miliberlin — COMMENTED** [review]
Can you set up redirects/aliases for the links you changed, so the user doesn't hit a 404?

**Comment by Dgiordano33** [comment]
Did not mean to touch the docs on this branch - will revert!

**Comment by Dgiordano33** [comment]
> Can you set up redirects/aliases for the links you changed, so the user doesn't hit a 404?

Did you see any specifically besides in the Doc changes? I think all the links in Learn should be redirected.

**Review by miliberlin — APPROVED** [review]
LGTM 👍

**Pull request merged**